app/services/orphaned_profiles_cleaner.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Service for cleaning up orphaned profiles
+class OrphanedProfilesCleaner
+  class << self
+    def run!
+      # No need to run this in an empty DB
+      return if Profile.canonical(false).none?
```

Comment:
What's the reason for this optimization? Ephemeral test run would not run the part of the code after this return.

Reply:
It's not really an optimization, the query below would probably fail on an empty DB + also it has no point to run it on an empty DB.